onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -89,12 +108,31 @@ cd diffusers
 pip install -e .
 ```
 
+### Setup Environment (ROCm)
+
+It is recommended that the users should run the model with ROCm 5.4 or newer and Python 3.9. Note that Windows is not
+supported for ROCm at the moment.
+
+```
+conda create -n py39 python=3.9
+conda activate py39
+pip install -r requirements.txt
+```
+
+AMD GPU version of torch build can be installed from [AMD Radeon repo](https://repo.radeon.com/rocm/manylinux/rocm-rel-5.4/),
+user need to download the whl file and `pip install <file.whl>` manually. Or directly from https://download.pytorch.org/whl/rocm5.4.2 via
+```
+pip download torch==1.1.1 --index-url https://download.pytorch.org/whl/rocm5.4.2
```

Comment:
Change to torch==2.0.0? Could not find a version that satisfies the requirement torch==1.1.1+rocm5.4.2